spec/defines/monit_monitor_spec.rb

```diff
@@ -75,13 +75,34 @@
       it 'should compile' do
         should contain_file('/etc/monit.d/monit-monitor-group.conf').with_content(
           "check process monit-monitor-group with pidfile /var/run/monit.pid\n" +
-          "  start program = \"/etc/init.d/monit-monitor-group start\"\n" +
-          "  stop program  = \"/etc/init.d/monit-monitor-group stop\"\n" +
+          "  start program = \"/usr/sbin/invoke-rc.d monit-monitor-group start\"\n" +
+          "  stop program  = \"/usr/sbin/invoke-rc.d monit-monitor-group stop\"\n" +
```

Comment:
These specs should have been changed when the template was changed. Oh well, fixed now.